src/systemctl/systemctl.c

```diff
@@ -283,6 +288,8 @@ static int systemctl_help(void) {
                "                         'us': 'Day YYYY-MM-DD HH:MM:SS.UUUUUU TZ\n"
                "                         'utc': 'Day YYYY-MM-DD HH:MM:SS UTC\n"
                "                         'us+utc': 'Day YYYY-MM-DD HH:MM:SS.UUUUUU UTC\n"
+               "     --read-only         Create read-only bind mount\n"
+               "     --mkdir             Create directory before bind mounting, if missing\n"
```

Comment:
"before bind-mounting" (The verb should be with a dash, the noun is ok without.)

Reply:
fixed